src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.HandleDictionary.cs

```diff
@@ -83,6 +84,13 @@ public static partial class JsonSerializer
 
                     // An object or another enumerator requires a new stack frame.
                     state.Push(elementClassInfo, value);
+                    var extensionDataAttribute = jsonPropertyInfo.PropertyInfo?.GetCustomAttribute(typeof(JsonExtensionDataAttribute));
+                    if (options.DictionaryKeyPolicy != null && extensionDataAttribute == null 
+                                                             && state.Current.ExtensionDataStatus != ExtensionDataWriteStatus.Writing)
```

Comment:
nit: remove extra spacing before the last `&&` condition

Reply:
Strange, I use Rider on Linux, it suggested me this style.